src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -35,16 +36,18 @@
         OutputSpec.newBuilder().addPrint("%s").addPrint(RESPONSE_PLACEHOLDER).build());
   }
 
-  static OutputView toView(OutputSpec config, MethodContext context) {
+  static OutputView toView(OutputSpec config, MethodContext context, SampleValueSet valueSet) {
     Runnable once =
         new Runnable() {
           boolean ran;
 
           @Override
           public void run() {
-            if (ran) {
-              throw new IllegalArgumentException("only one field of OutputSpec may be set");
-            }
+            Preconditions.checkArgument(
```

Comment:
We should test this. I'm fine with adding a `TODO(pongad)` for now if we're moving quickly for parallel development and things may change, but I don't want this to be untested for too long.